### ISSUE TITLE
Native Reverse Geocoder

### ISSUE THREAD

**Issue by arthursp** [ionic-team/ionic-native#1852]
**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request

**Current behavior:**
When using nativeGeocoder.reverseGeocode, the result is instance of NativeGeocoderReverseResult, and the interface is : 

`export interface NativeGeocoderReverseResult {
    /**
     * The street.
     */
    street: string;
    /**
     * The house number.
     */
    houseNumber: string;
    /**
     * The postal code.
     */
    postalCode: string;
    /**
     * The city.
     */
    city: string;
    /**
     * The district.
     */
    district: string;
    /**
     * The country name.
     */
    countryName: string;
    /**
     * The country code.
     */
    countryCode: string;
}`

But in the result object there is : 

`{"administrativeArea": "XXXXX", "countryCode": "XXXX", "countryName": "XXXX", "locality": "XXXXX", "postalCode": "1235", "subAdministrativeArea": "XXXX", "subThoroughfare": "1", "thoroughfare": "Rue XXXXX"}`

**Expected behavior:**

having the interface mapped to the correct attribute names.

**Steps to reproduce:**

Use this in a component ( after loaded each needed modules )

`this.geolocation.getCurrentPosition().then((resp) => {
                this.nativeGeocoder.reverseGeocode(resp.coords.latitude, resp.coords.longitude).then( ...`

**Related code:**

```
 */
export interface NativeGeocoderReverseResult {
    /**
     * The street.
     */
    street: string;
    /**
     * The house number.
     */
    houseNumber: string;
    /**
     * The postal code.
     */
    postalCode: string;
    /**
     * The city.
     */
    city: string;
    /**
     * The district.
     */
    district: string;
    /**
     * The country name.
     */
    countryName: string;
    /**
     * The country code.
     */
    countryCode: string;
}
```

**Other information:**

To fix it change the interface with the correct attributes

**package.json info:**

```json
insert the content here
```

**Comment by gabriellecozart** [comment]
Was about to leave the same issue with all the same information. Please fix.

**Comment by gabriellecozart** [comment]
Here is a temp fix if anyone needs it: 

Create a custom interface
```
export interface NativeGeocoderResultModel {
  subAdministrativeArea: string,
  postalCode: number,
  locality: string,
  subLocality: string,
  subThoroughfare: string,
  countryCode: string,
  countryName: string,
  administrativeArea: string,
  thoroughfare: string
}
```

Then create an object using that interface and set it equal to the parsed, stringified result
```
let newResult: NativeGeocoderResultModel = JSON.parse(JSON.stringify(result));
```

Then grab the new result and manipulate it however you would like.

**Comment by monichols** [comment]
@gabriellecozart YOU ARE A LIFE SAVER! Thanks girl!

**Comment by gabriellecozart** [comment]
@monichols Glad I could help!

**Comment by patrickr** [comment]
ran into this.. looks like it's an easy fix Will submit PR shortly

**Comment by patrickr** [comment]
turns out there is already a PR for this #1840

**Comment by iget-master** [comment]
Could someone take a look at this? The fix is already done on #1840, someone could merge it?

**Comment by vijaychauhanssn** [comment]
when i am starting this cmd: ionic cordova build android --proud

Typescript Error
Property 'countryName' does not exist on type 'NativeGeocoderReverseResult[]'.

let country = this.toaster.create({
  message: res.countryName,
  duration: 4000
Typescript Error
Property 'street' does not exist on type 'NativeGeocoderReverseResult[]'.
{
   let str : string   = `The reverseGeocode address is ${result.street} in ${result.countryCode}`;
   resolve(str);
Typescript Error
Property 'countryCode' does not exist on type 'NativeGeocoderReverseResult[]'.

{
   let str : string   = `The reverseGeocode address is ${result.street} in ${result.countryCode}`;
   resolve(str);
Typescript Error
Property 'latitude' does not exist on type 'NativeGeocoderForwardResult[]'.
C:/Users/Prateek/best/src/providers/geocoder/geocoder.ts
{
   let str : string   = `The coordinates are latitude=${coordinates.latitude} and longitude=${coordinates.longitude}`;
   resolve(str);


Property 'longitude' does not exist on type 'NativeGeocoderForwardResult[]'.


{
   let str : string   = `The coordinates are latitude=${coordinates.latitude} and longitude=${coordinates.longitude}`;
resolve(str);
